jetty-util/src/test/java/org/eclipse/jetty/util/MultiMapTest.java

```diff
@@ -471,7 +471,10 @@ public void testToString()
 
         mm.putValues("food", "apple", "cherry", "raspberry");
 
-        assertEquals("{color=red, food=[apple, cherry, raspberry]}", mm.toString());
+        String expected1 = "{color=red, food=[apple, cherry, raspberry]}";
+        String expected2 = "{food=[apple, cherry, raspberry], color=red}";
+        String actual = mm.toString();
+        assertTrue(actual.equals(expected1) || actual.equals(expected2));
```

Comment:
The correct way is ... `assertThat(actual, anyOf(is(expected1), is(expected2));`
As that will produce a meaningful error message when it fails.